assets/app/view/game/borrow_train.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+require 'view/game/actionable'
+
+module View
+  module Game
+    class BorrowTrain < Snabberb::Component
```

Comment:
Can you grab a piccy of the view so that it's easier to review?

Reply:
Yeah, I just need to run a test game first that doesn't use the cheating private revenues so I can use it as a test case too

Reply:
![image](https://user-images.githubusercontent.com/2993555/113464187-cd45ac80-93f8-11eb-86df-f93a7cdada51.png)